src/logic/calculate.js

```diff
@@ -24,34 +39,41 @@ const calculate = ({ total, next, operation }, bName) => {
       } if (bName === '.') {
```

Comment:
whenever `bName === .`  we should be able to return `0.` not `.`
I think the app doesn't look professional like that right?

Screenshot from your app below 👇
![image](https://user-images.githubusercontent.com/31897434/73010494-d4475700-3e12-11ea-82ed-d239f6699b39.png)